tools/tools.go

```diff
@@ -0,0 +1,11 @@
+//go:build tools
+
+package tools
+
+import (
+	_ "github.com/cosmos/gogoproto/protoc-gen-gocosmos"
+	_ "github.com/golang/protobuf/protoc-gen-go"
+	_ "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway"
+	_ "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger"
+	_ "github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2"
```

Comment:
Why need v2?

Reply:
@colmazia what's this file for?

Reply:
it is auto generated from ignite to generate go files from proto